static/docs/commands-reference/status.md

```diff
@@ -56,7 +56,16 @@ differ are listed. For each item listed, either the file name or the checksum is
 shown, and additionally a status word is shown describing the change:
 
 - For the local workspace:
-  - _changed_ means the named file has changed
+  - _changed deps_ means the DVC-file has changed dependencies
+  - _changed outs_ means the DVC-file has changed outputs
+  - _changed checksum_ means actual checksum of the DVC-file doesn't match the
+    one specified in it in DVC-files
+  - _always changed_ means that this is a special DVC-file with no dependencies,
+    which is considered always changed
+  - _new_ means the DVC-file has been created but not added into local cache
+  - _modified_ means the dependecies and output is changed in the DVC-file
+  - _deleted_ means the DVC-file has been deleted from local cache
+  - _not in cache_ means the DVC-file no longer exist in local cache
```

Comment:
so, this is not exactly correct presentation of those states.

first 4 mean that state of the file itself

last 4 mean the state of the output or dependency

We need to change the way we present them.

Please, take the get-started repo, do some simple experiments to get those states, see the pattern.